src/components/reports/reportSummary/reportSummaryDetails.tsx

```diff
@@ -119,8 +121,8 @@ const ReportSummaryDetailsBase: React.SFC<ReportSummaryDetailsProps> = ({
           content={t(
             `${perspective}_dashboard.total_cost_tooltip`,
             perspective === DashboardPerspective.ocp
-              ? { infrastructureCost, derivedCost }
-              : { infrastructureCost, markupCost }
+              ? { infrastructureCost, supplementaryCost }
+              : { infrastructureCost, supplementaryCost }
```

Comment:
We were showing derived and markup costs on different pages (e.g., Ocp on cloud showed markup). Just want to ensure markup is no longer available and they're both supplementary now?

See https://github.com/project-koku/koku-ui/blob/ffbcf10cf807d5d001d4b92ec372154638fac127/src/locales/en.json#L646

For the Ocp overview, the tooltip that shows:

<img width="493" alt="Screen Shot 2020-03-17 at 2 35 06 PM" src="https://user-images.githubusercontent.com/17481322/76889728-840dd380-685c-11ea-9902-353aeaf95f9a.png">

For the Ocp on cloud overview (now "All infrastructure filtered by Ocp cost"), the tooltip that shows:

<img width="442" alt="Screen Shot 2020-03-17 at 2 35 29 PM" src="https://user-images.githubusercontent.com/17481322/76889847-b5869f00-685c-11ea-938d-de45b20acd3c.png">

Reply:
If markup is removed, then we don't need that `perspective === DashboardPerspective.ocp` code

Reply:
So based on our revision to the cost terminology markup is not removed but can be part of both infrastructure and supplementary: https://docs.google.com/document/d/1hnwS0uwPuAKw75QyE2QEl8IZcczaQredMvDY10UVhLk/edit?usp=sharing

I didn't make adjustments to the perspectives, because I think the code flow needs to change more to pass the perspecive to the `runReport` method so that you can properly choose whether you want to use the following APIs:
- /reports/openshift/costs/
- /reports/openshift/infrastructures/all/costs/         (openshift on all infrastructures)
- /reports/openshift/infrastructures/aws/costs/      (openshift on all aws)
- /reports/openshift/infrastructures/azure/costs/      (openshift on all azure)

^ plus the corresponding APIs for storage and instance-types

The intent of the PR is to allow a bridge where you can update from here.

Reply:
Spoke with @nlcwong and this is expected.

In that case, we don't need that `perspective === DashboardPerspective.ocp` code